components/Icons/src/index.scss

```diff
@@ -4,6 +4,6 @@
   width: 1em;
   height: 1em;
   font-size: 1.5rem;
-  fill: currentcolor;
+  fill: currentColor;
```

Comment:
I think lowercase is preferred nowadays https://developer.mozilla.org/en-US/docs/Web/CSS/color_value#currentcolor_keyword